packages/editor/src/utils/media-upload/index.js

```diff
@@ -1,7 +1,12 @@
+/**
+ * External dependencies
+ */
+import { v4 as uuid } from 'uuid';
```

Comment:
this is throwing a linting error for some reason even though I see it used elsewhere:

`error  'uuid' should be listed in the project's dependencies. Run 'npm i -S uuid' to add it  import/no-extraneous-dependencies`

Reply:
It‘s not listed in `packages/editor/package.json`

Reply:
oh, thanks! I thought I had tried running the install command in that directory. I'll try again.

Reply:
Oh, now I remember: I got an error trying to add it "npm error ERESOLVE unable to resolve dependency tree".  Installing with `--legacy-peer-deps` seems to have worked though.